test/jdk/java/lang/RuntimeTests/RuntimeExitLogTest.java

```diff
@@ -64,31 +64,80 @@ public static void main(String[] args) throws InterruptedException {
         System.exit(status);
     }
 
+    /**
+     * Generate a regular expression pattern that match the expected log output for a Runtime.exit() call.
+     * The pattern includes the method call stack trace and the exit status.
+     * @param status the exit status passed to the Runtime.exit() call
+     * @return the regex pattern as a string
+     */
+    private static String generateStackTraceLogPattern(int status) {
+        return "(?s)^.+ java\\.lang\\.Shutdown logRuntimeExit\\n" +
+                ".*: Runtime\\.exit\\(\\) called with status: " + status + "\\n" +
+                "java\\.lang\\.Throwable: Runtime\\.exit\\(" + status + "\\)\\n" +
+                "\\s+at java\\.base/java\\.lang\\.Shutdown\\.logRuntimeExit\\(Shutdown\\.java:\\d+\\)\\n" +
+                "\\s+at(?: .+)";
```

Comment:
Testing the exact logging message is too brittle/too exact.
It should only be testing for the specific variable information that is unique to the case under test.
It becomes burdensome to update tests when some small change occurs in a message.

Reply:
The test should verify that the stack traces required for troubleshooting are logged correctly.
It avoid verifying line numbers likely to change frequently.